src/app/pages/credentials/backup-credentials/forms/cloud-credentials-form.component.ts

```diff
@@ -1244,26 +1244,8 @@ export class CloudCredentialsFormComponent implements FormConfiguration {
       function: () => {
         this.entityForm.loader.open();
         this.entityForm.error = '';
-        const attributes: any = {};
         const value = _.cloneDeep(this.entityForm.formGroup.value);
-        this.beforeSubmit(value);
```

Comment:
Please don't skip the call `beforeSubmit` function.
If OAuth URL is null, should delete value `client_id` and `client_secret`.

Reply:
Weird that I removed it. Fixed.